src/dmd/access.d

```diff
@@ -189,22 +189,16 @@ bool checkAccess(Loc loc, Scope* sc, Expression e, Dsymbol d)
     if (!e)
         return false;
 
-    if (auto tc = e.type.isTypeClass())
+    if (auto ta = e.type.isTypeAggregate())
```

Comment:
Honestly, I prefer the old version of the code, it's much easier to distinguish between the `class` case and the `struct` case.

Reply:
Sure, I tried to remove repetition here, but I can revert this in particular.

Reply:
Done.